addons/website_sale/static/src/scss/website_sale_backend.scss

```diff
@@ -23,6 +23,7 @@
             padding-bottom: 100%;
             > img {
                 position: absolute;
+                height: 100%;
```

Comment:
not sure :/
Without:
![image](https://user-images.githubusercontent.com/4848797/86461284-0ea21200-bd2a-11ea-8002-81fdd669f23d.png)


With:
![image](https://user-images.githubusercontent.com/4848797/86461238-f8945180-bd29-11ea-8f3a-553e752fe45f.png)


While for thumbnail of slider in frontend, I found it could to user cover instead of contain
For preview in backend, and for for picture in frontend, I will prefer to keep ratio

Reply:
same for width 200px in preview 

![image](https://user-images.githubusercontent.com/4848797/86461413-4315ce00-bd2a-11ea-829d-3e5612599f39.png)

vs

![image](https://user-images.githubusercontent.com/4848797/86461454-5032bd00-bd2a-11ea-9b84-7024e41a9fa5.png)